src/header/header.css

```diff
@@ -0,0 +1,13 @@
+header {
```

Comment:
Not a blocker but do we want all `<header>` elements to be red and `2.5em` tall?